scripts/install_julia.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+# Latest Julia version (this needs to be manually updated, periodically)
+JULIA_LATEST=1.6-latest
+
+JULIA_VERSION=${1:-${JULIA_VERSION:-latest}}
+
+if [ "$JULIA_VERSION" = "latest" ]; then
+    JULIA_VERSION=$JULIA_LATEST
+fi
+JULIA_MAJOR_VERSION=${JULIA_VERSION:0:3}
+
+# Download and extract Julia. We'll use a common /opt/julia/ location regardless of version 
+wget https://julialang-s3.julialang.org/bin/linux/x64/$JULIA_MAJOR_VERSION/julia-$JULIA_VERSION-linux-x86_64.tar.gz -O /opt/julia.tar.gz
+mkdir /opt/julia
+tar zxvf /opt/julia.tar.gz -C /opt/julia --strip-components 1
+rm -rf /opt/julia.tar.gz
+
+# Add to path
+echo 'export PATH="/opt/julia/bin:$PATH"' >> .bashrc
+source .bashrc
```

Comment:
Checking other scripts, I think there is room for modification in the way the PATH is set.

https://github.com/rocker-org/rocker-versioned2/blob/3b41d37f6d4c008b26f5cc8c0083f960b86be6cc/scripts/install_pyenv.sh#L22-L24

How to set environment variables is currently being discussed in https://github.com/rocker-org/rocker-versioned2/issues/151#issuecomment-834574069, but the above description seems to be the standard at this time.

I am not an expert in these scripts or PATH settings, so if this comment is incorrect, please feel free to ignore it.